pom.xml

```diff
@@ -0,0 +1,168 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.github.giis-uniovi</groupId>
+    <artifactId>retorch</artifactId>
+    <version>1.1.1-SNAPSHOT</version>
```

Comment:
Change the version number to 1.0.0
After merge, I will create a release with this number (as prerelease) to refer it from zenodo.